assets/app/views/services.html

```diff
@@ -15,8 +15,16 @@
         <div>Port: {{service.spec.port}}</div>
         <div>Container port: {{service.spec.containerPort}}</div>
         <div>Protocol: {{service.spec.protocol}}</div>
-        <div>Selectors:<span ng-repeat="(selectorLabel, selectorValue) in service.spec.selector"> {{selectorLabel}}={{selectorValue}}</span></div>
-      </div>      
+        <div>Selectors: <span ng-if="!service.spec.selector"><em>none</em></span><span ng-repeat="(selectorLabel, selectorValue) in service.spec.selector"> {{selectorLabel}}={{selectorValue}}<span ng-show="!$last">, </span></span></div>
+        <div>Routes: <span ng-if="(!routesByService[service.metadata.name]) || (routesByService[service.metadata.name] | hashSize) == 0"><em>none</em></span><span ng-if="(routesByService[service.metadata.name] | hashSize) > 0" ng-repeat="(routeName, route) in routesByService[service.metadata.name]">
```

Comment:
Also, you don't need the `if > 0` test... repeating over an empty value will just never call the repeat